tutorials/lidar/create-canopy-height-model-in-R.Rmd

```diff
@@ -83,8 +83,10 @@ surface model (DSM) and the digital terrain model (DTM).
 library(raster)
 library(rgdal)
 
-# set working directory to data folder
-#setwd("pathToDirHere")
+# set working directory to ensure R can find the file we wish to import and where
+# we want to save our files. Be sure to move the download into your working directory!
+wd="~/Desktop/LIDAR_Tutorial/" #This will depend on your local environment
```

Comment:
Change as per today's discussion

Reply:
I made this update and changed "LiDAR" to "lidar" per Tristan's convention.